src/lib/Microsoft.Health.Events/EventConsumers/Service/EventBatchingService.cs

```diff
@@ -263,6 +263,8 @@ private void LogDataFreshness(string partitionId, string triggerReason, IEnumera
             // If no events were flushed for the partition (eg: trigger reason is ThresholdWaitReached - due to receival of MaxTimeEvent), then use the current timestamp.
             var eventTimestampLastProcessed = events?.Any() ?? false ? events.Last().EnqueuedTime.UtcDateTime : DateTime.UtcNow;
             _logger.LogMetric(EventMetrics.EventTimestampLastProcessedPerPartition(partitionId, triggerReason), double.Parse(eventTimestampLastProcessed.ToString("yyyyMMddHHmmss")));
+
+            _logger.LogMetric(EventMetrics.EventFreshnessDelayPerPartition(partitionId), DateTime.UtcNow.Subtract(eventTimestampLastProcessed).TotalMinutes);
```

Comment:
I think by convention metrics we expose to customers that are time based should be in milliseconds.

Reply:
@wi-y to confirm but it looks like based on https://1dsdocs.azurewebsites.net/articles/shoebox/shoebox-metrics.html#guidelines-for-units valid units for time should either be milliseconds or seconds.  Suggestion change to total seconds (that should be precise enough, I don't think we need milliseconds).

Reply:
For Latency metrics I believe only Milliseconds are supported based on a later section of the doc, but it may be worth asking the internal team if Seconds are supported since that seems easier to read for a user.

Reply:
Sent an email to the Shoebox Core group mentioned in the article asking for an exception to the rule that latency metrics must have milliseconds as their unit.